cla_backend/apps/cla_eventlog/management/commands/find_and_delete_old_cases.py

```diff
@@ -22,20 +22,23 @@ def handle(self, *args, **kwargs):
         instance = FindAndDeleteCasesUsingCreationTime()
         cases = instance.get_eligible_cases()
         django_command = sys.argv[1]
-        if args and args[0] == "delete":
-            if len(args) > 1 and args[1] == "no-input":
-                instance.run()
-            elif django_command == "test":
+
+        if django_command == "test":  # If command is run in test
+            if args and args[0] == "delete":
                 instance.run()
             else:
+                return cases
+
+        if django_command == "find_and_delete_old_cases":  # If command is run in terminal
```

Comment:
I would remove this condition and just have it as the `else` to your previous one.
If we imagine a situation where I wrote a new management command called "do_cleanup_stuff.py" and inside that I did:
```
call_command("find_and_delete_old_cases")
```
this command wouldn't run properly. That's undesirable. But I do like the idea.

Reply:
Thanks. Done